scripts/util/resolve-types.js

```diff
@@ -29,15 +29,15 @@ module.exports = (shapesWithoutNames, { inputShapes, outputShapes }) => {
     decoder: `${jsonDecode}.bool`,
     jsonEncoder: `${jsonEncode}.bool`,
     queryEncoderType: 'toString',
-    queryEncoder: base => `AWS.Encode.addOneToQueryArgs toString "${base}"`,
+    queryEncoder: base => `AWS.Encode.addOneToQueryArgs toString "${upCam(base)}"`,
```

Comment:
At least for SQS, these need to be up-camel case (i.e. first letter uppercase). Will need to verify if this is true for all query protocol based services